code/_onclick/observer.dm

```diff
@@ -14,7 +14,7 @@
 
 	// Otherwise jump
 	else if(A.loc)
-		loc = get_turf(A)
+		Move(get_turf(A.loc))

```

Comment:
This is going to cause a whole lot of bugs because a lot of stuff assumes that ghosts don't called crossed/etc